File: objc/Romeo/Romeo_ios/C0015_SENDCLEAR/C0015_SENDCLEAR.m
#import <Foundation/Foundation.h>

#import "C0015_SENDCLEAR.h"

@implementation C0015_SENDCLEAR : NSObject

-(void) bad {
    NSString * const LOCAL_USER_URL = @"http://localhost:8080/bar/user";
    NSMutableURLRequest *req = [NSMutableURLRequest requestWithURL:[NSURL URLWithString:LOCAL_USER_URL]];
    [[[NSURLSession sharedSession] dataTaskWithRequest: req] resume];
}

-(void) good {
    NSString * const LOCAL_USER_URL = @"https://localhost:8080/bar/user";
    NSMutableURLRequest *req = [NSMutableURLRequest requestWithURL:[NSURL URLWithString:LOCAL_USER_URL]];
    [[[NSURLSession sharedSession] dataTaskWithRequest: req] resume];
}

@end

File: swift/src/main/swift/C0015_SENDCLEAR/C0015_SENDCLEAR.swift
package swift.C0015_SENDCLEAR

import Foundation

func bad() {
    let LOCAL_USER_URL = "http://localhost:8080/bar/user"
    let req : NSMutableURLRequest = NSMutableURLRequest(URL:NSURL(string:LOCAL_USER_URL))
    let conn : NSURLConnection = NSURLConnection(req:req, delegate:self)
}

func good() {
    let LOCAL_USER_URL = "https://localhost:8080/bar/user"
    let req : NSMutableURLRequest = NSMutableURLRequest(URL:NSURL(string:LOCAL_USER_URL))
    let conn : NSURLConnection = NSURLConnection(req:req, delegate:self)
}